config/packages/security.yaml

```diff
@@ -19,6 +16,10 @@ security:
             id: sylius.shop_user_provider.email_or_name_based
         sylius_api_shop_user_provider:
             id: sylius.shop_user_provider.email_or_name_based
+        sylius_api_admin_and_shop_user_provider:
```

Comment:
`sylius_api_admin_and_shop_user_provider` -> `sylius_api_user_provider`? For even better unification 🏇